### ISSUE TITLE
October 20  ( Version 6fef21c ) Agent decline mission error

### ISSUE THREAD

**Issue by Chryst** [ISeeDEDPpl/Questor#226]
I don't know if it is already fixed or no, but after it decline a mission happend that error. is that normal ?

Or is already fixed in the more new versions ?

2:19:58 [Cleanup] Found a window that needs 'yes' chosen...
2:19:58 [Cleanup] Content of modal window (HTML): [If you decline a mission before 2012.10.23 03:38 you will lose
tandings with this agent, as well as his corp and faction.  If you lose enough standings, you will no longer be able t
o the agent.  Are you sure you would like to decline this mission?]
2:20:24 [Questor] Wallet Balance Has Not Changed in [ 2 ] minutes.
2:20:38 [AgentInteraction] WaitForMission: Unable to find mission from that agent (yet?) : AgentInteraction.AgentId
3016893] regular Mission AgentID [XXXXXX]
2:20:39 [AgentInteraction] Opening Journal Window: waiting [21sec]
2:21:00 [AgentInteraction] WaitForMission: Unable to find mission from that agent (yet?) : AgentInteraction.AgentId
3016893] regular Mission AgentID [XXXXXX]
2:21:01 [AgentInteraction] Opening Journal Window: waiting [19sec]
2:21:21 [AgentInteraction] WaitForMission: Unable to find mission from that agent (yet?) : AgentInteraction.AgentId
3016893] regular Mission AgentID [XXXXXX]
2:21:22 [AgentInteraction] Opening Journal Window: waiting [16sec]
2:21:24 [Questor] Wallet Balance Has Not Changed in [ 3 ] minutes.
2:21:38 [AgentInteraction] WaitForMission: Unable to find mission from that agent (yet?) : AgentInteraction.AgentId
3016893] regular Mission AgentID [XXXXXX]
2:21:39 [AgentInteraction] Opening Journal Window: waiting [14sec]
2:21:54 [AgentInteraction] WaitForMission: Unable to find mission from that agent (yet?) : AgentInteraction.AgentId
3016893] regular Mission AgentID [XXXXXX]
2:21:54 [AgentInteraction] Opening Journal Window: waiting [18sec]

**Comment by Chryst** [comment]
Seems solved with the last version, we reopen if the error really happend again.

**Comment by Chryst** [comment]
With the last version the error is little different, Simply it get stucked when it decline missions.

**Comment by I3Questor** [comment]
Yes I have found this as well.

**Comment by ISeeDEDPpl** [comment]
this should be fixed. if not please provide a console log when it occurs.

**Comment by Chryst** [comment]
Ok, now i try it. I know only with the version of yesterday ( 3e853ebab3 ) when that error happend, all the clients become closed with the xml file not found error, and with the log folder with 4 fonder of the char have caused in error, not in his char folder but moved to the main log folder, not in the log/charname folder.

That disalignment of the folders cause a crash for all the chars becouse they become unable to find their right xml.

I think all the chars give a disalignment of all the folders so they become unable to find their right xml file.

**Comment by ISeeDEDPpl** [comment]
what you describe has nothing to do with this issue. the ".xml" error you describe was caused by your client being disconnected and the Charactername no longer being filled in, it was literally empty so that when Questor tried to refresh the settings is tried to load an XML file named ".xml" instesad of "myCharacterName.xml". I have corrected questor to detect when this happens (the eve disonnect) and restart questor. At the moment questor restarts with a 30-60sec randomish timedcommand in the uplink,. if your internet connection or vpn goes down for longer than this then the restart will potentially be unsuccessful.
